new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItDedicatedMode.java

```diff
@@ -0,0 +1,403 @@
+ */
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+@DisplayName("Test Operator and WebLogic domain with Dedicated set to true")
+@IntegrationTest
+class ItDedicatedMode {
+  // namespace constants
+  private static String opNamespace = null;
+  private static String domain1Namespace = null;
+  private static String domain2Namespace = null;
+
+  // domain constants
+  private final String domainUid = "domain1";
+  private final String clusterName = "cluster-1";
+  private final int replicaCount = 2;
+  private final String adminServerPodName = domainUid + "-" + ADMIN_SERVER_NAME_BASE;
+  private final String managedServerPodPrefix = domainUid + "-" + MANAGED_SERVER_NAME_BASE;
+
+  // operator constants
+  private static HelmParams opHelmParams;
+  private static String opServiceAccount;
+  private static final String domainNamespaceSelectionStrategy = "Dedicated";
+
+  private static LoggingFacade logger = null;
+
+  /**
+   * Get namespaces for operator and domain2.
+   *
+   * @param namespaces list of namespaces created by the IntegrationTestWatcher by the
+   *                   JUnit engine parameter resolution mechanism.
+   */
+  @BeforeAll
+  public static void initAll(@Namespaces(2) List<String> namespaces) {
+    logger = getLogger();
+    // get a unique operator namespace
+    logger.info("Getting a unique namespace for operator");
+    assertNotNull(namespaces.get(0), "Namespace list is null");
+    opNamespace = namespaces.get(0);
+
+    // in the dedicated mode, the operator only manages domains in the operator's own namespace
+    domain1Namespace = opNamespace;
+
+    // get a new unique domainNamespace
+    logger.info("Assigning a unique namespace for Domain");
+    assertNotNull(namespaces.get(1), "Namespace list is null");
+    domain2Namespace = namespaces.get(1);
+
+    logger.info("Installing and verifying operator");
+    opServiceAccount = opNamespace + "-sa";
+    opHelmParams =
+        new HelmParams().releaseName(OPERATOR_RELEASE_NAME)
+            .namespace(opNamespace)
+            .chartDir(OPERATOR_CHART_DIR);
+  }
+
+  @AfterAll
+  public void tearDownAll() {
+  }
+
+  /**
+   * Set "dedicated" to true and set the domainNamespaces to something that does not contain the operator's namespace,
+   * make sure that the domains in the operator's target namespaces do not come up
+   * because when dedicated is set to true, the operator's domainNamespaces value is ignored.
+   * 1) Create CRD.
+   * 2) Install an Operator with namespace=op-ns, domainNamespaces=wls-ns
+   *    and domainNamespaceSelectionStrategy=Dedicated.
+   * 3) Verify the Operator is up and running.
+   * 4) Create WebLogic Domain its namespace=wls-ns.
+   * 5) Verify that domain whose namespace = wls-ns does not come up.
+   */
+  @Test
+  @Order(1)
+  @DisplayName("Set dedicated to true and verify that a domains not deployed in operator's namespace doesn't come up")
```

Comment:
that domains not deployed or that a domain not deployed

Reply:
fixed